packages/in_app_purchase/in_app_purchase_storekit/example/shared/RunnerTests/TranslatorTests.m

```diff
@@ -458,4 +458,40 @@ - (void)testSKPaymentTransactionConvertToPigeon {
   XCTAssertEqual(paymentTransaction.transactionIdentifier, msg.transactionIdentifier);
 }
 
+- (void)testSKProductResponseCovertToPigeon {
+  SKProductsResponseStub *response =
+      [[SKProductsResponseStub alloc] initWithMap:self.productResponseMap];
+  SKProductsResponseMessage *responseMsg =
+      [FIAObjectTranslator convertProductsResponseToPigeon:response];
+
+  XCTAssertEqual(responseMsg.products.count, 1);
+  XCTAssertEqual(responseMsg.invalidProductIdentifiers.count, 0);
+
+  SKProductMessage *productMsg = responseMsg.products[0];
+
+  XCTAssertEqualObjects(productMsg.price, @"1");
```

Comment:
nit: all of these asserts are against values that are being set in `setUp` right? It's kind of far from here.  Adding a comment wouldn't hurt.